src/applications/disability-benefits/2346/actions/index.js

```diff
@@ -0,0 +1,38 @@
+import * as Sentry from '@sentry/browser';
+
+import { getVeteranInformation } from '../api';
+import {
+  FETCH_VETERAN_INFORMATION,
+  FETCH_VETERAN_INFORMATION_FAILURE,
+} from '../constants';
+
+export const fetchVeteranInformation = data => ({
+  type: FETCH_VETERAN_INFORMATION,
+  data,
+});
+
+export const fetchVeteranInformationFailure = () => ({
+  type: FETCH_VETERAN_INFORMATION_FAILURE,
+});
+
+export const getVeteranInformationData = () => async dispatch => {
+  try {
+    const data = await getVeteranInformation();
+    const veteranInformation = {
+      veteranFullName: data.formData.veteranFullName,
+      gender: data.formData.gender,
+      dateOfBirth: data.formData.dateOfBirth,
+      veteranAddress: data.formData.veteranAddress,
+      email: data.formData.email,
+    };
+    dispatch(fetchVeteranInformation(veteranInformation));
+  } catch (error) {
+    dispatch(
+      { FETCH_VETERAN_INFORMATION_FAILURE },
```

Comment:
You need to dispatch this as a type.  If not, you will get a failure.  `VM1456:1 Uncaught (in promise) Error: Actions may not have an undefined "type" property. Have you misspelled a constant?`

Reply:
@bradl3yC Making this get dispatched using the action creator.